services/services.go

```diff
@@ -61,6 +63,8 @@ type ApiService struct {
 	Mode             ApiType
 	DatabaseService  dboperations.IDbOperations
 	TemplateRenderer template.TemplateRender
+	replicated       replicated.IReplicated
```

Comment:
can we rename this to `Replicated` as all the contents of this struct are capital

Reply:
Done